examples/basic_openshift/conjur-java-api-example.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: conjur-cli-example
+  labels:
+    app: conjur-cli-example
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: conjur-cli-example
+  template:
+    metadata:
+      name: conjur-cli-example
+      labels:
+        app: conjur-cli-example
+    spec:
+      serviceAccountName: default
+      containers:
+      - image: cyberark/conjur-authn-k8s-client:latest
+        imagePullPolicy: IfNotPresent
+        name: authenticator
+        env:
+          - name: CONJUR_VERSION
+            value: "5"
+          - name: CONTAINER_MODE
+            value: sidecar
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: MY_POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: CONJUR_AUTHN_URL
+            value: https://conjur-oss.ofira-conjur-test.svc.cluster.local/authn-k8s/myauthenticator
```

Comment:
Can you please remove references of your name and use more generic names in this demo?